model/host/db.go

```diff
@@ -998,6 +997,52 @@ func AggregateLastContainerFinishTimes() ([]FinishTime, error) {
 
 }
 
+func AddVolumeToHost(h *Host, newVolume VolumeAttachment) error {
```

Comment:
I think you can do these both as methods on *Host, and you just pass the pointer reciever to the findAndModify... 

I think the current api implications of making this a function (and the fact that it modifies the host you pass it anyway, is confusing.

Reply:
I'm good to make it a method, I just didn't see any others in this file. I still think this should be in db.go given the FindAndModify though.